userguide/config.toml

```diff
@@ -1,21 +1,22 @@
 baseURL = "/"
 title = "Docsy Example"
 
-enableRobotsTXT = true
-
-# Hugo allows theme composition (and inheritance). The precedence is from left to right.
-theme = ["docsy"]
-
-# Will give values to .Lastmod etc.
-enableGitInfo = true
-
 # Language settings
 contentDir = "content/en"
 defaultContentLanguage = "en"
 defaultContentLanguageInSubdir = false
 # Useful when translating.
 enableMissingTranslationPlaceholders = true
 
+enableRobotsTXT = true
+
+# set docsy as theme, either locally or as hugo module
+theme = ["docsy"]
+# theme = ["github.com/google/docsy", "github.com/google/docsy/dependencies"]
+
+# Will give values to .Lastmod etc.
+enableGitInfo = true
+
```

Comment:
If we decide to keep this change, then I have one nit: please move this block of lines back to where it was, that is, starting in line 4 -- this will help keep blame history clean.

Reply:
Changing the order of blocks here was done on purpose, and I would like to keep it. See my [PR](https://github.com/google/docsy-example/pull/162) submitted to the `docsy-example`project for an explanation.

Reply:
Thanks for the link to the explanation.